redi-connect-front/src/components/organisms/SideMenu.scss

```diff
@@ -16,15 +26,44 @@
       padding: 0.8rem 0;
       margin-right: -1rem;
 
+      @include mobile() {
+        display: inline-flex;
+        align-items: center;
+        flex-direction: column;
+        font-size: $size-6;
+        letter-spacing: 0;
+        text-transform: none;
+        padding: 0;
+      }
+
+
       &--active {
         font-weight: $weight-bold;
         color: $black;
         border-right: 5px solid $redi-blue;
+
+        & .side-menu__icon g {
```

Comment:
simpler would be just g (does to job)

`& .side-menu__icon g {` -> `g {`

Reply:
done